lib/rb_shift/client.rb

```diff
@@ -73,9 +76,12 @@ def projects(update = false)
 
     def execute(command, **opts)
       command = "oc --server=\"#{@url}\" --token=\"#{@token}\" #{command}  #{unfold_opts opts}"
-      Open3.popen3 command do |_, _, stderr, wait_thrd|
-        err_code = wait_thrd.value
-        raise InvalidCommandError unless stderr.gets.nil? && err_code.success?
+      log.info command
```

Comment:
Logging command with info level is not a good idea.
First of all the output of library will be too much verbose and it can be considered as security issue since the token will be written to output as well.